README.md

```diff
@@ -17,12 +17,20 @@ A drop-in replacement for the standard Zipkin HTTP collector that writes to the
 Stackdriver Trace service.
 
 ### Configuration
+Typically, sampling is done by the Tracer prior to sending data to the Collector, so the Collector
```

Comment:
Yeah, that probably makes more sense. I had forgotten zipkin-server exposed this as an environment variable.